### DISCUSSION TITLE
Problem with latex rendering

### DISCUSSION THREAD

**Discussion by budinero** [requarks/wiki#4254] [Help / Questions]
Hi, 

I have wiki.js installed with docker, following the basic guide. I have enabled only one of the latex rendering options. 

When writing equations in markdown, I can see the equation properly rendered on the preview panel. But when the page is saved, only white spaces are shown in the wiki instead of the equations. Figures and tables are working properly. 

I tried with mathjax and with katex. 

Wiki.js version 2.5.201 . 

What can I try?

Thanks!

**Comment by clarencesees** [comment]
@budinero did you ever get this sorted? I'm having a similar issue.

**Comment by vnpenguin** [comment]
I have the same issue with 2.5.306.
When I see you posted in 2021. No response in 2025???

**Reply by Podolyakofs** [reply]
same here with mathjax
2.5.308

**Reply by BPowell76** [reply]
I have seen issues with using Latex when making pages. Pretty sure it's a bug, but when including Latex in a markdown table, for example, if I don't put other latex command/macro after a close `}` the expression fails to render properly. I just started putting `\space` at the end as a hack/workaround.

This currently doesn't work for me in the wiki:

```md
| col1 | col2 |
| :---:| :---:|
| $\frac{a}{b}$ | aaa |
```

This does:

```md
| col1 | col2 |
| :---:| :---:|
| $\frac{a}{b}\space$ | aaa |
```
